### PR TITLE
Change the build requires so it can be built for 3.3

### PR DESCRIPTION
**Pull request by efajardo**
@bbockelm  since you want this for 3.3 but 3.3 does not have `hadoop-libhdfs-devel`, I made this change that should be sufficient.

### CODE REVIEW

**Comment by bbockelm** [comment]
Upstream packages should use version numbers, not the release numbers.  Can you push an update and bump the version to 2.0.2?

**Comment by efajardo** [comment]
@bbockelm allright done.

File: rpm/xrootd-hdfs.spec [hunk 1]
```diff
@@ -65,6 +65,9 @@ rm $RPM_BUILD_ROOT%{_bindir}/xrootd_hdfs_envcheck
 %{_includedir}/XrdHdfs.hh
 
 %changelog
+* Tue Apr 03 2018 Edgar Fajardo <efajardo@physics.ucsd.edu> - 2.0.2-1
+- Change the build requires so it can be built for 3.3 (SOFTWARE-3179)
```
Comment (edquist) [inline rpm/xrootd-hdfs.spec]: Is this the right ticket number?  SOFTWARE-3179 is about Internet2 packages

File: rpm/xrootd-hdfs.spec [hunk 2]
```diff
@@ -12,7 +12,7 @@ Source0: %{name}-%{version}.tar.gz
 BuildRequires: xrootd-devel >= 1:4.6
 BuildRequires: xrootd-server-devel >= 1:4.6
 BuildRequires: cmake
-BuildRequires: hadoop-libhdfs-devel >= 2.0.0+545-1.cdh4.1.1
+BuildRequires: /usr/include/hdfs.h
```
Comment (edquist) [inline rpm/xrootd-hdfs.spec]: Ah, that's a bummer that hadoop 2.0.0 doesn't have a separate libhdfs-devel package.

Just a note here, apparently we used to care about the version requirement, but apparently now `hdfs.h` is only provided by versions of hadoop we still use, so it should be fine now as-is without the version requirement.

File: rpm/xrootd-hdfs.spec [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Name: xrootd-hdfs
-Version: 2.0.1
+Version: 2.0.2
```
Comment (edquist) [inline rpm/xrootd-hdfs.spec]: Bumping the Version number is in line with what @bbockelm requested, so I won't ask you to change it back.  But normally a spec-only change would only warrant bumping the Release number.  Bundling the spec file in the upstream repo perhaps makes it a bit trickier to de-couple packaging-only changes from non-packaging changes in releases, but `osg-build` does have support for tags like `v2.0.1-2`, where the `-2` tag indicates only spec changes were made.

**Comment by efajardo** [comment]
@edquist good catch I fixed the ticket number

**Comment by bbockelm** [comment]
LGTM also.  I'm going to merge and tag a release.  @efajardo - I'll leave it to you to retry the build.  If we screw it up ... well, we'll make a 2.0.3.

**Comment by efajardo** [comment]
That is great @bbockelm since I do not have access to make releases

**Pull request merged**